src/app/public/modules/vertical-tabset/vertical-tabset.component.ts

```diff
@@ -103,9 +109,12 @@ export class SkyVerticalTabsetComponent implements OnInit, AfterViewChecked, OnD
   @Output()
   public activeChange = new EventEmitter<number>();
 
-  @ViewChild('contentWrapper')
+  @ViewChild('groupContainerWrapper')
   public tabGroups: ElementRef;
 
+  @ViewChild('contentContainerWrapper')
+  public contentWrapper: ElementRef;
```

Comment:
Need this to be public? I know the other view children are, but if this can be private that would be ideal.